contacts/emails_method_setting.py

```diff
@@ -0,0 +1,38 @@
+# gmail settings
```

Comment:
define these in the root settings of your django project, not in the specific app.

Reply:
no this is unnecessary file by mistake push sorry